### PR TITLE
Fix package name in pip upgrade instructions

### PR DESCRIPTION
**Pull request by davedittrich**
Fixes https://github.com/ansible/ansible-lint/issues/2622 and adds support for Conda virtual environments.

### CODE REVIEW

**Review by ssbarnea — APPROVED** [review]

File: src/ansiblelint/config.py [hunk 1]
```diff
@@ -192,27 +192,31 @@ def parse_ansible_version(stdout: str) -> tuple[str, str | None]:
 
 def in_venv() -> bool:
     """Determine whether Python is running from a venv."""
-    if hasattr(sys, "real_prefix"):
+    if (
+        hasattr(sys, "real_prefix")
+        or os.environ.get('CONDA_EXE', None) is not None
+    ):
```
Comment (apatard) [inline src/ansiblelint/config.py]: If the commit is about package name, why is this change here ?

**Comment by apatard** [comment]
While you're at it, could you fix also the issue of always printing the " Upgrade by running: " message please ? (see my comment in #2622 )

File: src/ansiblelint/config.py (hunk 1 above)
Comment (davedittrich) [reply src/ansiblelint/config.py]: Because it looked from the path comparison logic like the code didn't take into consideration use of Conda as a virtual environment (though it is). I wasn't running in VScode, so maybe "real_prefix" is in `sys` with Conda?

Comment (apatard) [reply src/ansiblelint/config.py]: Oh, I was not commenting the change by itself (since I'm not sure to understand it), just that imho, it was deserving its own commit.

**Comment by isuftin** [comment]
@apatard  - Same https://github.com/ansible/ansible-lint/issues/2622#issuecomment-1294068923
